### PR TITLE
Compute API: Rename Compute "create_volume_snapshot" 

### PR DESCRIPTION
**Pull request by dhague**
to "create_snapshot" to avoid confusion with Volume API snapshots

### CODE REVIEW

**Comment by gildub** [comment]
Ok, just a rename in lib/fog/openstack/docs/compute.md needed too.

File: lib/fog/openstack/compute.rb [hunk 1]
```diff
@@ -231,73 +231,75 @@ class Mock
...
+                'injected_file_path_bytes'    => 256,
+                'injected_files'              => 5,
+                'metadata_items'              => 128,
+                'floating_ips'                => 10,
+                'instances'                   => 10,
+                'key_pairs'                   => 10,
+                'gigabytes'                   => 5000,
+                'volumes'                     => 10,
+                'cores'                       => 20,
+                'ram'                         => 51200
```
Comment (dhague) [inline lib/fog/openstack/compute.rb]: Won't fix - see #107

**Comment by gildub** [comment]
Please check inline comments: "A rename in lib/fog/openstack/docs/compute.md is needed".
LGTM besides above.

**Comment by dhague** [comment]
ah, good catch. thanks.
One problem with the hundreds of Hound comments is that it's hard to see the more useful comments from human reviewers...

**Comment by gildub** [comment]
No worries

**Comment by gildub** [comment]
Great, thanks

**Pull request merged**